src/com/facebook/buck/apple/project_generator/ProjectGenerator.java

```diff
@@ -1503,6 +1506,27 @@ private PBXNativeTarget generateBinaryTarget(
     return target;
   }
 
+  private ImmutableList<String> getExcludesForModulesUnderTests(
+      TargetNode<? extends CxxLibraryDescription.CommonArg, ?> targetNode) {
+    ImmutableList.Builder<String> testingOverlayBuilder = new ImmutableList.Builder<>();
+    visitRecursivePrivateHeaderSymlinkTreesForTests(
+        targetNode,
+        (nativeNode, headerVisibility) -> {
```

Comment:
Nit: I think it would be more descriptive for `nativeNode` to be named `libBeingTested`.

Reply:
👍